shell/browser/atom_paths.cc

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2013 GitHub, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include "shell/browser/atom_paths.h"
+
+#include "base/files/file_path.h"
+#include "base/files/file_util.h"
+#include "base/path_service.h"
+#include "chrome/common/chrome_paths.h"
+#include "shell/browser/browser.h"
+
+#if defined(OS_MACOSX)
+#include "shell/browser/atom_paths_mac.h"
+#endif
+
+#if defined(USE_X11)
+#include <memory>
+#include "base/environment.h"
+#include "base/nix/xdg_util.h"
+#endif
+
+namespace electron {
+
+namespace {
+
+#if defined(USE_X11)
+void GetLinuxAppDataPath(base::FilePath* path) {
+  std::unique_ptr<base::Environment> env(base::Environment::Create());
+  *path = base::nix::GetXDGDirectory(env.get(), base::nix::kXdgConfigHomeEnvVar,
+                                     base::nix::kDotConfigDir);
+}
+#endif
+
+}  // namespace
+
+// We can not use PathService with a provider !
```

Comment:
I think this comment paragraph is out-of-date, referring to a previous version of the PR?